build/int.cloudbuild.yaml

```diff
@@ -21,15 +21,23 @@ steps:
       - 'TF_VAR_org_id=$_ORG_ID'
       - 'TF_VAR_folder_id=$_FOLDER_ID'
       - 'TF_VAR_billing_account=$_BILLING_ACCOUNT'
+
+  # Sometimes API activation can be bit flaky therefor ensure that at least 3 mins (180s) are passed between
+  # prepare_environment and the first terraform apply (init-all runs parallel to reduce waste of time)
+  - id: wait for api activation
+    waitFor:
+      - prepare
+    name: 'alpine:latest'
+    args: ['sleep', '$_API_ACTIVATION_SECONDS_WAIT']
+
   - id: init-all
     waitFor:
       - prepare
```

Comment:
If we change this to depend on `wait for api activation` that will let us skip adding this individually to each step. Something like prepare->wait for api activation-> init-all -> other steps

Reply:
The Idea was we have a fixed waiting time and that waiting time we also use for init-all.
Happy to change if you don't like the idea

Reply:
The Idea was we have a fixed waiting time and that waiting time we also use for init-all.
Happy to change if you don't like the idea

Reply:
Gotcha! SGTM